docs/1.0. Overview.md

```diff
@@ -31,6 +31,54 @@ Inputs and Outputs both have a number of Channels available for use. An Input re
 
 The relationship between Input channels and Output channels is defined by the "Map". Where a given Input is routable to a given Output, it MUST be possible to route any of the Input's channels to any channel of the Output. Output channels that do not have any Input channels routed to them are said to be "parked".
 
+The following shows an example of an IS-08 Map between one Input with a large number of channels (for example from a 64-channel MADI interface) and two eight-channel Outputs, represented as a logical wiring diagram and as a routing matrix:
+
+![terminology example](images/map-examples/terminology.png "Terminology example")
+
+Note that:
+
+- The eight Output A channels are routed as a block to the eight Input A channels. This could be due to hardware limitations on those channels, which can be signalled as a constraint in IS-08 (see below).
+- Channels 7 and 8 of each Output are routed to the same Input A channels.
+- Channels 5 and 6 of Output B are parked.
+
+
+The above Map is represented in JSON in the API as below. See [Behaviour](4.0.%20Behaviour.md) for more details.
+
+```json
+"map":{
+  "Output A":{
+    "chan 1" :{
```

Comment:
Utterly trivial niggle: whitespace is a bit inconsistent in this JSON example (probably is in examples too, since I think it came from there...). Normally I think we have `"prop": {` (rather than with any whitespace before the colon or none after).

Reply:
D'oh, we realised I missed the obvious mistake we made that the `"map"` properties should be output ids like `"outA"` (no spaces allowed), and the properties of that must be the stringized channel indexes, like `"0"`, etc. not channel names.

Reply:
@garethsb-sony that's now corrected, and I've updated all the mapping examples to show both input and output channel indexes. Or indices if you prefer.